GETTING_STARTED.md

```diff
@@ -1,50 +1,50 @@
+1. Commit and push your changes.
 
 ## Submitting a pull request for review
 
-1. Once you’re ready to submit your changes for review, head back over to the [main repository](https://github.com/EthanThatOneKid/acmcsuf.com) and click on Pull Requests → New pull request
-1. Your pull request should contain
-   1. A clear and descriptive title (Please do not use “Fixed #123”)
-   1. A clear description that describes, in detail, what you changed and how
-   1. A link to the issue that you resolved (if applicable)
-   1. Any screenshots if your change affects something visible on the website or that you believe would help the reviewer understand your changes
-   1. Examples of good PRs
-      1. [https://github.com/EthanThatOneKid/acmcsuf.com/pull/477](https://github.com/EthanThatOneKid/acmcsuf.com/pull/477)
-      1. [https://github.com/EthanThatOneKid/acmcsuf.com/pull/469](https://github.com/EthanThatOneKid/acmcsuf.com/pull/469)
-      1. [https://github.com/EthanThatOneKid/acmcsuf.com/pull/401](https://github.com/EthanThatOneKid/acmcsuf.com/pull/401)
-   1. Examples of bad PRs
-      1. [https://github.com/EthanThatOneKid/acmcsuf.com/pull/359](https://github.com/EthanThatOneKid/acmcsuf.com/pull/359)
-      1. [https://github.com/EthanThatOneKid/acmcsuf.com/pull/357](https://github.com/EthanThatOneKid/acmcsuf.com/pull/357)
-1. Request a code review from someone by first contacting them and then assigning them to your PR using the “Reviewers” panel
-1. Address any feedback from your reviewers if applicable
-1. Once your changes have been approved, you have permission to merge your changes, and your changes should be merged with a **SQUASH** and **NOT** a merge commit
+1. Once you’re ready to submit your changes for review, head back over to the [main repository](https://acmcsuf.com/code) and click on Pull Requests → New pull request.
+1. Your pull request should contain.
+   1. A clear and descriptive title (Please do not use "Fixed #123").
+   1. A clear description that describes, in detail, what you changed and how.
+   1. A link to the issue that you resolved (if applicable).
+   1. Any screenshots if your change affects something visible on the website or that you believe would help the reviewer understand your changes.
+   1. Examples of good PRs:
+      1. [https://acmcsuf.com/pull/477](https://github.com/EthanThatOneKid/acmcsuf.com/pull/477)
+      1. [https://acmcsuf.com/pull/469](https://github.com/EthanThatOneKid/acmcsuf.com/pull/469)
+      1. [https://acmcsuf.com/pull/401](https://github.com/EthanThatOneKid/acmcsuf.com/pull/401)
+   1. Examples of bad PRs:
+      1. [https://acmcsuf.com/pull/359](https://github.com/EthanThatOneKid/acmcsuf.com/pull/359)
+      1. [https://acmcsuf.com/pull/357](https://github.com/EthanThatOneKid/acmcsuf.com/pull/357)
+1. Request a code review from someone by first contacting them and then assigning them to your PR using the "Reviewers" panel.
```

Comment:
Should we mention that reviewers have to review changes within 48 hours?

Reply:
Good idea. This is a fitting location to mention that common courtesy is to review within 24 hours if possible. The author of the PR should also alert their reviewer in some way other than GitHub to make them aware that there is a PR awaiting their review.

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/865/commits/82d1a77b696bccfc0117a29fea76c71565501cdd>.